modules/administration-guide/pages/installing-che-using-storage-classes.adoc

```diff
@@ -53,10 +53,17 @@ spec:
   devEnvironments:
     storage:
       pvc:
-        # keep blank unless you need to use a non default storage class for workspace PVC(s)
-        storageClass: 'workspace-storage'
+        # the required storage class field to modify depends on whether the per-user or per-workspace PVC strategy is being used
+        pvcStrategy: 'per-user'
+        perUserStrategyPvcConfig:
+          # keep blank unless you need to use a non default storage class for workspace PVC(s)
+          storageClass: 'workspace-storage'
+        perWorkspaceStrategyPvcConfig:
+          # keep blank unless you need to use a non default storage class for workspace PVC(s)
+          storageClass: 'workspace-storage'        
```

Comment:
```suggestion
        pvcStrategy: 'per-user' <1>
        perUserStrategyPvcConfig: <2>
          storageClass: 'workspace-storage'
        perWorkspaceStrategyPvcConfig: <3>
          storageClass: 'workspace-storage'        
```
<1> he required storage class field to modify depends on whether the per-user or per-workspace PVC strategy is being used
<2> keep blank unless you need to use a non default storage class for workspace PVC(s)
<3> keep blank unless you need to use a non default storage class for workspace PVC(s)

Reply:
I still haven't updated the page on storage classes (going to take a break for today) but I wanted to try and answer this question already.

The `storageClass` field can be set to the name of a [Kubernetes storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/) that exists on the cluster where Che is installed. IIRC, storage class names follow the [following naming convention](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names), though this detail is probably outside the scope of Che docs. 

I can figure out how to explain this detail in the callout the next time I work on this PR.